tenseal/serialization.h

```diff
@@ -0,0 +1,56 @@
+#ifndef TENSEAL_SERIALIZATION_H_
+#define TENSEAL_SERIALIZATION_H_
+
+#include <string>
+
+#include "seal/seal.h"
+
+namespace tenseal {
+
+/**
+ * Saves a SEAL object to a string.
+ * Compatible SEAL types: Ciphertext, Plaintext, SecretKey, PublicKey,
+ *GaloisKeys, RelinKeys.
+ **/
+template <class T>
+std::string SEALSerialize(const T& sealobj) {
+    std::stringstream stream;
+    sealobj.save(stream);
+
+    return stream.str();
+}
```

Comment:
I think `EncryptionParameters` object is also compatible with this function, isn't it?